statsmodels/robust/scale.py

```diff
@@ -49,6 +49,46 @@ def mad(a, c=Gaussian.ppf(3/4.), axis=0, center=np.median):
     return np.median((np.abs(a-center)) / c, axis=axis)
 
 
+def iqr(a, c=Gaussian.ppf(3/4) - Gaussian.ppf(1/4), axis=0, center=np.median):
```

Comment:
Don't you need a 2* in the definition of c?

Reply:
Nope, that's the right c

Reply:
I missed the - because I was on mobile.  It is the same as 2*ppf of 3/4 as well.